src/qt/forms/debugwindow.ui

```diff
@@ -1100,13 +1100,39 @@
                 </widget>
                </item>
                <item row="2" column="0">
+                <widget class="QLabel" name="peerNetworkLabel">
+                 <property name="toolTip">
+                  <string>The network protocol this peer is connected through: IPv4, IPv6, Onion, I2P, or CJDNS.</string>
```

Comment:
The suggestion for a follow-up is to remove untranslatable network names from the translatable tooltip string.